tensilelite/Tensile/Source/client/include/TypedId.hpp

```diff
@@ -108,6 +108,7 @@ namespace Tensile
 #ifdef TENSILE_USE_BF16
     using TypedGemm_B_B_S = TypedGemm<BFloat16, BFloat16, BFloat16, BFloat16, float, float>;
     using TypedGemm_B_S_S = TypedGemm<BFloat16, BFloat16, float, float>;
+    using TypedGemm_HH_B_H_S = TypedGemm<Half, Half, Half, Half, float, float, BFloat16>;
 #endif // TENSILE_USE_BF16
 #ifdef TENSILE_USE_FP8_BF8
```

Comment:
I'm not sure of this naming, maybe @jichangjichang you have any comments?

Reply:
should be H_B_H_S